resources/js/Contexts/DraftGalleriesContext.tsx

```diff
@@ -46,6 +48,18 @@ export const DraftGalleriesContextProvider = ({ children }: ProviderProperties):
         await loadWalletDrafts();
     };
 
+    useEffect(() => {
+        const isMyGalleryDraftPage = route().current("my-galleries", { draft: true });
+
+        if (isTruthy(drafts) && drafts.length === 0 && isMyGalleryDraftPage) {
+            router.visit(
+                route("my-galleries", {
+                    draft: false,
+                }),
```

Comment:
think you can also just redirect to `route("my-galleries")` without specifying the "draft" query param